src/extensions/surveys.tsx

```diff
@@ -219,3 +118,752 @@ export function generateSurveys(posthog: PostHog) {
         callSurveys(posthog, false)
     }, 3000)
 }
+
+const defaultBackgroundColor = '#eeeded'
+
+export function QuestionHeader({
```

Comment:
Generally speaking I would already want this to be a bit cleaner as a codebase. No reason I can see why not to put this in the `surveys` subfolders something like:

```
/surveys/components/QuestionHeader.tsx
/surveys/constants.ts // for things like colors
```

I think it's worth doing already as it will become messy quickly and harder to review in the future.

Reply:
Not following the reasoning tbh? You need to import preact, therefore you don't put them in different files?